### PR TITLE
Drop legacy /oapi cmd tests

### PR DESCRIPTION
**Pull request by sttts**
Preparations for https://github.com/openshift/origin/pull/21775

### CODE REVIEW

**Comment by sttts** [comment]
Preparations for https://github.com/openshift/origin/pull/21775

File: test/extended/builds/build_pruning.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ var _ = g.Describe("[Feature:Builds][pruning] prune builds based on settings in
 		g.It("[Conformance] buildconfigs should not have a default history limit set when created via the legacy api", func() {
 
 			g.By("creating a build config with the legacy api")
-			err := oc.Run("create").Args("-f", legacyBuildConfig, "--raw=/oapi/v1/namespaces/"+oc.Namespace()+"/buildconfigs").Execute()
+			err := oc.Run("create").Args("-f", legacyBuildConfig, "--raw=/api/v1/namespaces/"+oc.Namespace()+"/buildconfigs").Execute()
```
Comment (deads2k) [inline test/extended/builds/build_pruning.go]: Yeah, this change won't work.  We should make a commit deleting tests like this and let people know they're gone.

File: test/cmd/authentication.sh [hunk 2]
```diff
@@ -73,10 +73,10 @@ os::cmd::expect_success_and_text "oc get projects --token='${allescalatingpowers
 os::cmd::expect_success_and_text "oc policy can-i --list --token='${allescalatingpowerstoken}' -n '${project}'" 'get.*pods'
 
 accesstoken="$(oc process -f "${OS_ROOT}/test/testdata/authentication/scoped-token-template.yaml" TOKEN_PREFIX=access SCOPE=user:check-access USER_NAME="${username}" USER_UID="${useruid}" | oc create -f - -o name | awk -F/ '{print $2}')"
-os::cmd::expect_success_and_text "curl -k -XPOST -H 'Content-Type: application/json' -H 'Authorization: Bearer ${accesstoken}' '${API_SCHEME}://${API_HOST}:${API_PORT}/oapi/v1/namespaces/${project}/localsubjectaccessreviews' -d @${OS_ROOT}/test/testdata/authentication/localsubjectaccessreview.json" '"kind": "SubjectAccessReviewResponse"'
+os::cmd::expect_success_and_text "curl -k -XPOST -H 'Content-Type: application/json' -H 'Authorization: Bearer ${accesstoken}' '${API_SCHEME}://${API_HOST}:${API_PORT}/api/v1/namespaces/${project}/localsubjectaccessreviews' -d @${OS_ROOT}/test/testdata/authentication/localsubjectaccessreview.json" '"kind": "SubjectAccessReviewResponse"'
```
Comment (deads2k) [inline test/cmd/authentication.sh]: the referenced file needs to be updated.

File: test/extended/builds/build_pruning.go (hunk 1 above)
Comment (sttts) [reply test/extended/builds/build_pruning.go]: true, api groups. But we can switch to the specific API group and drop legacy paths.

**Pull request closed without merging**